docs/tools/replace.sh

```diff
@@ -36,25 +41,73 @@ function get_volumes() {
             -o go-template='{{range .items}}{{.metadata.name}}{{ " " | print }}{{end}}'
 }
 
-# usage: get_pod_name <volume>
+# usage: get_pod_name <volume-id>
 function get_pod_name() {
     # shellcheck disable=SC2016
     kubectl get directpvvolumes "${1}" \
             -o go-template='{{range $k,$v := .metadata.labels}}{{if eq $k "directpv.min.io/pod.name"}}{{$v}}{{end}}{{end}}'
 }
 
-# usage: get_pod_namespace <volume>
+# usage: get_pod_namespace <volume-id>
 function get_pod_namespace() {
     # shellcheck disable=SC2016
     kubectl get directpvvolumes "${1}" \
             -o go-template='{{range $k,$v := .metadata.labels}}{{if eq $k "directpv.min.io/pod.namespace"}}{{$v}}{{end}}{{end}}'
 }
 
+# usage: get_node_name <drive-id>
+function get_node_name() {
+    # shellcheck disable=SC2016
+    kubectl get directpvdrives "${1}" \
+            -o go-template='{{range $k,$v := .metadata.labels}}{{if eq $k "directpv.min.io/node"}}{{$v}}{{end}}{{end}}'
+}
+
+# usage: is_uuid input
+function is_uuid() {
+    [[ "$1" =~ ^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$ ]]
+}
+
+# usage: get_drive_id <node> <drive-name>
+function get_drive_id() {
+    if [ -z "${1}" ]; then
+        id_err="node argument must be provided for drive name"
+        return
+    fi
+    # shellcheck disable=SC2207
+    drive_ids=( $(get_drive_ids "${1}" "${2}") )
+    if [ "${#drive_ids[@]}" -eq 0 ]; then
+        id_err="drive '${2}' on node '${1}' not found"
+        return
+    fi
+    if [ "${#drive_ids[@]}" -gt 1 ]; then
+        id_err="duplicate drive ids found for '${2}'"
```

Comment:
You exit right here after printing error.  There is no need to store. If you feel function name would confuse, use `must_get_drive_id`